source/components/Layout/Layout.tsx

```diff
@@ -9,24 +9,26 @@ interface ILayout {
   children: React.ReactNode;
   id?: string;
   title: string;
+  titleOnly: boolean;
```

Comment:
```suggestion
  titleOnly?: boolean;
```

this way you don't need to set the titleOnly prop to false on all components, it sets it to false by default